src/RoverPhoto/RoverPhoto.scss

```diff
@@ -0,0 +1,28 @@
+.topDiv{
```

Comment:
Can we name this file the same as it's counterpart (RoverTopPhoto) please and also separate out the css for RoverTopPhoto and Curiosity.tsx into separate files

Reply:
Christian, we had separate files for rover top and curiosity. we merged them because, this css will be common to all the rovers. And named it not as rover top, but as rover as it is css for top and all images. Do you still think we need to do the above changes?